data/free worlds start.txt

```diff
@@ -1615,6 +1780,124 @@ mission "FW Pirates 4.1"
+			`	A confused roar breaks out. A number of senators cry out their disapproval, but an equal number of senators thump their armrests or clap in approval. The older senator purses his lips, rubs his chin, and furrows his brow. Ariana scowls. JJ scowls back at her, sits down, and leans over to you. "Now we'll see how much backbone the senate truly has."
+				goto sentence
+			label hero
+			`	"Indeed," says the senator. "And I bet it wasn't long after the battle that you realized saving the galaxy isn't that easy."`
+			`	JJ shakes his head as he leans forward to respond. "Senator, this was never about being a hero to the galaxy to Captain <last>. This was about being a hero to those slaves. Many of them have been waiting for generations to escape to a chance at a better life."`
+			`	The old senator blinks. "And you think Captain <last> can provide them a better life?"`
+			`	JJ shrugs. "They're free to find it themselves, just as we are. The free slaves seem determined to become their own heroes from now on, but I would think that the Senate and Council of the Free Worlds should be happy to help them find their way if ever they ask."`
+			label sentence
+			`	A third senator speaks up, a woman whose name card identifies her as Sara, from Skymoot. "That will be all, Captain <last>. The Senate will confer and decide whether we want your continued services."`
+			`	You are led back out to the lobby, where you wait for hours. Finally, JJ comes and leads you to  Senator Sara's office. `
+			`	The senator gestures for you both to sit down and pours you a drink. "Captain <last>, you've put us in a bind. News of your actions is spreading like wildfire, and much of the public is firmly behind you and your actions. Many of us personally feel great admiration for your actions on Greenrock. Obviously, we can't condemn someone who has freed so many slaves and destroyed such a serious threat. From a practical standpoint, we are almost certain to need you and your ship in the coming battles with the Republic.`
+			`	Senator Sara sips her drink, then continues. "However, we are a government, and governments work on the rule of law. Your actions were morally correct, but legally, you've made a hell of a mess for us. Our government is still working out basic principles and the right balance of power. We cannot just allow ship captains in our forces to simply do as they please. Therefore you must have some kind of public reprimand and punishment.`
+			`	"Officially, we have decided that you will be suspended for your disobedience without pay. Separately, and rather unofficially, we have adjusted our clandestine operations budget to pay you a regular retainer. We will need your services again, and we hope you will come quickly when we contact you."`
+			goto goodbye
+			label goodbye
```

Comment:
Which part? The "clandestine ops" paycheck? The idea there is that they're disciplining the player officially, while quietly acknowledging that he/she did a good thing. The senate realizes that the player is a major asset, and they don't want to totally alienate him/her, because they see the need for someone who can fight and win against all the odds in the future. 

Would a straight-up exoneration be preferable? Or are you suggesting that I simply leave things the same way as in the original Senate Hearing mission?